homeworks/src/main/java/hw36/ConsoleProgram.java

```diff
@@ -0,0 +1,44 @@
+package hw36;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.logging.Logger;
+
+public class ConsoleProgram {
+    private static final Logger LOGGER = Logger.getLogger(ConsoleProgram.class.getName());
+
+    public static void startConsoleProgram(Cart cart, ProductRepository productRepository) {
+        try (BufferedReader reader = new BufferedReader(new InputStreamReader(System.in))) {
+            int choice;
+            do {
+                LOGGER.info("1. Додати товар до кошика");
+                LOGGER.info("2. Видалити товар з кошика");
+                LOGGER.info("3. Вийти");
+                LOGGER.info("Ваш вибір: ");
+                choice = Integer.parseInt(reader.readLine());
+                switch (choice) {
+                    case 1 -> {
+                        LOGGER.info("Введіть ID товару, який потрібно додати: ");
```

Comment:
Just for your information, as a user of your application, I'm now aware of an id of the product, but I know the name/description. No need to change anything, this is just some additional information + things to analyze.